hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenAccountWipeHandler.java

```diff
@@ -229,6 +230,9 @@ private ValidationResult validateSemantics(
         validateTrue(token.wipeKey() != null, ResponseCodeEnum.TOKEN_HAS_NO_WIPE_KEY);
 
         final var accountRel = TokenHandlerHelper.getIfUsable(accountId, tokenId, tokenRelStore);
+        if (token.hasKycKey()) {
```

Comment:
Where is this check in mono-service. I might have missed this 🤔

Reply:
[This test](https://github.com/hashgraph/hedera-services/pull/8611/files#diff-516656ed4ec511f5fe643accae195e3c0064b054aeb282b129c2b6f545a1facbR215) fails without the check. I accept the tests as a source of truth. I'm not very familiar with the mono repo, but it might be handled [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/models/TokenRelationship.java#L93) because on wipe we're changing balances as well?